jbot/src/main/java/me/ramswaroop/jbot/core/slack/models/Event.java

```diff
@@ -62,6 +65,8 @@
     private String domain;
     @JsonProperty("email_domain")
     private String emailDomain;
+    @JsonProperty("team")
+    private String teamId;
```

Comment:
@ramswaroop "teamId" will be part of all message event, but it might not be available as part of message event, when "subtype" of it is any of the following:
pinned_item, unpinned_item, message_changed and message_deleted